charts/bpa/values.yaml

```diff
@@ -66,6 +66,13 @@ bpa:
     i18n:
       locale: en
       fallbackLocale: en
+    runtimeVariables:
```

Comment:
I think naming this runtime variables is confusing as all entries here are. I would prefer something like _bpa.config.frontend.closeSidebar_

Reply:
Done